voxblox_ros/src/tsdf_server.cc

```diff
@@ -291,6 +293,13 @@ void TsdfServer::publishSlices() {
   tsdf_slice_pub_.publish(pointcloud);
 }
 
+void TsdfServer::publishPointclouds() {
+  // Combined function to publish everything we got.
```

Comment:
Not sure if this is the best method name for its purpose.

Reply:
"Everything we got" which is a pointcloud :) So all the stuff published by this function is a pointcloud message; some are surface pointclouds and some are pointclouds with all the points in the map.